### ISSUE TITLE
Multiple errors after updating to newest version.

### ISSUE THREAD

**Issue by Ookami313** [frozn/TipTac#78]
Updated to 22.04.15 and I'm receiving the following error when I log into the game. Thanks.

2x TipTac\ttCore.lua:712: bad argument #1 to 'unpack' (table expected, got nil)
[string "=[C]"]: ?
[string "@TipTac\ttCore.lua"]:712: in function `ApplySettings'
[string "@TipTac\ttCore.lua"]:1938: in function `?'
[string "@TipTac\ttCore.lua"]:556: in function <TipTac\ttCore.lua:556>
[string "=[C]"]: in function `LoadAddOn'
[string "@FrameXML\UIParent.lua"]:544: in function `UIParentLoadAddOn'
[string "@Rarity\Core\Collections-Collections.lua"]:63: in function `ScanToys'
[string "@Rarity\Core\Collections-Collections.lua"]:282: in function `ScanExistingItems'
[string "@Rarity\Core.lua"]:175: in function `DoEnable'
[string "@Rarity\Core.lua"]:151: in function <Rarity\Core.lua:150>
[string "=[C]"]: ?
[string "@DataStore\libs\AceAddon-3.0\AceAddon-3.0-13.lua"]:70: in function <DataStore\libs\AceAddon-3.0\AceAddon-3.0.lua:65>
...
[string "@PetTracker\libs\WildAddon-1.0\WildAddon-1.0-1.lua"]:56: in function <...dOns\PetTracker\libs\WildAddon-1.0\WildAddon-1.0.lua:53>
[string "=[C]"]: ?
[string "@AchieveIt\libs\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:29: in function <...eIt\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:25>
[string "@AchieveIt\libs\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:64: in function `Fire'
[string "@DataStore\libs\AceEvent-3.0\AceEvent-3.0-4.lua"]:120: in function <DataStore\libs\AceEvent-3.0\AceEvent-3.0.lua:119>
[string "=[C]"]: in function `LoadAddOn'
[string "@FrameXML\UIParent.lua"]:544: in function `UIParentLoadAddOn'
[string "@FrameXML\UIParent.lua"]:591: in function `CombatLog_LoadUI'
[string "@FrameXML\UIParent.lua"]:1456: in function <FrameXML\UIParent.lua:1337>
[string "=[C]"]: ?

Locals:
(*temporary) = nil
(*temporary) = "table expected, got nil"

**Comment by Ookami313** [comment]
Mousing over objects in the game, such as the mailbox, causes the following errors to occur as well.

15x TipTac\ttCore.lua:1045: Usage: GameTooltip:SetPoint("point" [, region or nil] [, "relativePoint"] [, offsetX, offsetY])
[string "=[C]"]: in function `SetPoint'
[string "@TipTac\ttCore.lua"]:1045: in function <TipTac\ttCore.lua:1010>
[string "@TipTac\ttCore.lua"]:1472: in function <TipTac\ttCore.lua:1466>
[string "=[C]"]: ?
[string "=[C]"]: in function `GameTooltip_SetDefaultAnchor'
[string "*:OnTooltipSetDefaultAnchor"]:1: in function <[string "*:OnTooltipSetDefaultAnchor"]:1>
[string "=[C]"]: ?

Locals:
(*temporary) = GameTooltip {
 0 = <userdata>
 SetQuestPartyProgress = <function> defined =[C]:-1
 OnBackdropLoaded = <function> defined @SharedXML\Backdrop.lua:144
 GetBackdropColor = <function> defined @SharedXML\Backdrop.lua:382
 SetHyperlink = <function> defined =[C]:-1
 SetUnit = <function> defined =[C]:-1
 SetAzeriteEssence = <function> defined =[C]:-1
 SetVoidDepositItem = <function> defined =[C]:-1
 SetVoidWithdrawalItem = <function> defined =[C]:-1
 ttSetBackdropBorderColorLocked = true
 SetSocketGem = <function> defined =[C]:-1
 ClearBackdrop = <function> defined @SharedXML\Backdrop.lua:274
 Show = <function> defined =[C]:-1
 SetPetAction = <function> defined =[C]:-1
 SetLootItem = <function> defined =[C]:-1
 FadeOut = <function> defined @TipTac\ttCore.lua:1453
 SetRecipeReagentItem = <function> defined =[C]:-1
 SetBuybackItem = <function> defined =[C]:-1
 textRight2Font = "GameTooltipText"
 SetWeeklyReward = <function> defined =[C]:-1
 SetMerchantItem = <function> defined =[C]:-1
 SetMerchantCostItem = <function> defined =[C]:-1
 Hide = <function> defined =[C]:-1
 BottomOverlay = <unnamed> {
 }
 SetHeirloomByItemID = <function> defined =[C]:-1
 SetItemKey = <function> defined =[C]:-1
 SetEnhancedConduit = <function> defined =[C]:-1
 ttIcon = <unnamed> {
 }
 GetEdgeSize = <function> defined @SharedXML\Backdrop.lua:180
 NineSlice = <unnamed> {
 }
 SetInventoryItem = <function> defined =[C]:-1
 TopEdge = <unnamed> {
 }
 ttCount = <unnamed> {
 }
 SetLootRollItem = <function> defined =[C]:-1
 SetCompanionPet = <function> defined =[C]:-1
 SetLFGDungeonReward = <function> defined =[C]:-1
 SetAuctionSellItem = <function> defined =[C]:-1
 SetBagItem = <function> defined =[C]:-1
 SetQuestCurrency = <function> defined =[C]:-1
 BottomRightCorner = <unnamed> {
 }
 SetBackdropColor = <function> defined @TipTac\ttCore.lua:1656
 LeftEdge = <unnamed> {
 }
 SetAuctionItem = <function> defined =[C]:-1
 SetCurrencyTokenByID = <function> defined =[C]:-1
 SetCurrencyByID = <function> defined =[C]:-1
 SetAzeriteEssenceSlot = <function> defined =[C]:-1
 ttSetIconTextureAndText = <function> defined @TipTacItemRef\ttItemRef.lua:169
 SetConduit = <function> defined =[C]:-1
 SetSpellBookItem = <function> defined =[C]:-1
 SetAction = <function> defined =[C]:-1
 textLeft1Font = "GameTooltipHeaderText"
 SetBackpackToken = <function> defined =[C]:-1
 SetExistingSocketGem = <function> defined =[C]:-1
 SetCurrencyToken = <function> defined @Altoholic\Services\Tooltip.lua:553
 SetGuildBankItem = <function> defined =[C]:-1
 SetVoidItem = <function> defined =[C]:-1
 SetBackdropBorderColor = <function> defined @TipTac\ttCore.lua:1664
 SetTrainerService = <function> defined =[C]:-1
 SetOwner = <function> defined =[C]:-1
 SetupTextureCoordinates = <function> defined @SharedXML\Backdrop.lua:206
 Center = <unnamed> {
 }
 TextRight1 = GameTooltipTextRight1 {
 }
 RightEdge = <unnamed> {
 }
 ClearLines = <function> defined =[C]:-1
 SetBackdrop = <function> defined @TipTac\ttCore.lua:1648
 BottomEdge = <unnamed> {
 }
 TopOverlay = <unnamed> {
 }
 GetBackdropBorderColor = <function> defined @SharedXML\Backdrop.lua:401
 GetBackdrop = <function> defined @SharedXML\Backdrop.lua:339
 SetUnitDebuff = <function> defined =[C]:-1
 SetItemByID = <function> defined =[C]:-1
 BottomLeftCorner = <unnamed> {
 }
 SetQuestItem = <function> defined =[C]:-1
 TopRightCorner = <unnamed> {
 }
 SetBorderBlendMode = <function> defined @SharedXML\Backdrop.lua:258
 SetToyByItemID = <function> defined =[C]:-1
 TopLeftCorner = <unnamed> {
 }
 HasBackdropInfo = <function> defined @SharedXML\Backdrop.lua:270
 backdropInfo = <table> {
 }
 SetInboxItem = <function> defined =[C]:-1
 shoppingTooltips = <table> {
 }
 ttGradient = <unnamed> {
 }
 SetLFGDungeonShortageReward = <function> defined =[C]:-1
 SetQuestLogItem = <function> defined =[C]:-1
 OnBackdropSizeChanged = <function> defined @

17x TipTac\ttCore.lua:661: attempt to perform arithmetic on field 'right' (a nil value)
[string "@TipTac\ttCore.lua"]:661: in function `SetPaddingVariables'
[string "@TipTac\ttCore.lua"]:1415: in function <TipTac\ttCore.lua:1413>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?

Locals:
self = TipTac {
 BottomLeftCorner = <unnamed> {
 }
 close = <unnamed> {
 }
 AddModifiedTip = <function> defined @TipTac\ttCore.lua:2001
 MirrorAnchorsSmart = <table> {
 }
 CURSOR_UPDATE = <function> defined @TipTac\ttCore.lua:455
 GetNearestPixelSize = <function> defined @TipTac\ttCore.lua:607
 GetBackdropColor = <function> defined @SharedXML\Backdrop.lua:382
 RightEdge = <unnamed> {
 }
 CVAR_UPDATE = <function> defined @TipTac\ttCore.lua:534
 ClearBackdrop = <function> defined @SharedXML\Backdrop.lua:274
 SetBackdropColorLocked = <function> defined @TipTac\ttCore.lua:774
 ADDON_LOADED = <function> defined @TipTac\ttCore.lua:1910
 SetBackdropBorderColorLocked = <function> defined @TipTac\ttCore.lua:789
 ResetBackdropBorderColorLocked = <function> defined @TipTac\ttCore.lua:804
 tipsToModify = <table> {
 }
 SetupTextureCoordinates = <function> defined @SharedXML\Backdrop.lua:206
 SendElementEvent = <function> defined @TipTac\ttCore.lua:423
 HasBackdropInfo = <function> defined @SharedXML\Backdrop.lua:270
 GetEdgeSize = <function> defined @SharedXML\Backdrop.lua:180
 TopEdge = <unnamed> {
 }
 ApplyUnitAppearance = <function> defined @TipTac\ttCore.lua:1172
 BottomRightCorner = <unnamed> {
 }
 padding = <table> {
 }
 LeftEdge = <unnamed> {
 }
 UI_SCALE_CHANGED = <function> defined @TipTac\ttCore.lua:541
 SetBackdropLocked = <function> defined @TipTac\ttCore.lua:767
 SetBackdropBorderColor = <function> defined @SharedXML\Backdrop.lua:414
 ApplyHooksToTips = <function> defined @TipTac\ttCore.lua:1733
 BottomEdge = <unnamed> {
 }
 SetBackdrop = <function> defined @SharedXML\Backdrop.lua:321
 GetBackdropBorderColor = <function> defined @SharedXML\Backdrop.lua:401
 GetBackdrop = <function> defined @SharedXML\Backdrop.lua:339
 playerLevel = 60
 SetBorderBlendMode = <function> defined @SharedXML\Backdrop.lua:258
 isColorBlind = false
 MirrorAnchors = <table> {
 }
 RegisterElement = <function> defined @TipTac\ttCore.lua:417
 CreatePushArray = <function> defined @TipTac\ttCore.lua:397
 PLAYER_LEVEL_UP = <function> defined @TipTac\ttCore.lua:474
 RemoveUnwantedLines = <function> defined @TipTac\ttCore.lua:1081
 AnchorFrameToMouse = <function> defined @TipTac\ttCore.lua:1051
 elements = <table> {
 }
 ClassColorMarkup = <table> {
 }
 Center = <unnamed> {
 }
 OnBackdropSizeChanged = <function> defined @SharedXML\Backdrop.lua:174
 ApplyHooksToLFGLFAVSFB = <function> defined @TipTac\ttCore.lua:1877
 AddLockingFeature = <function> defined @TipTac\ttCore.lua:1630
 GetUnitReactionIndex = <function> defined @TipTac\ttCore.lua:1111
 OnBackdropLoaded = <function> defined @SharedXML\Backdrop.lua:144
 ApplySettings = <function> defined @TipTac\ttCore.lua:676
 ApplyHooksToCFMLLSF = <function> defined @TipTac\ttCore.lua:1854
 TopLeftCorner = <unnamed> {
 }
 0 = <userdata>
 text = <unnamed> {
 }
 AddTargetedBy = <function> defined @TipTac\ttCore.lua:1137
 ApplyTipBackdrop = <function> defined @TipTac\ttCore.lua:890
 ApplyBackdrop = <function> defined @SharedXML\Backdrop.lua:286
 ReApplyAnchorTypeForMouse = <function> defined @TipTac\ttCore.lua:1059
 SetPadding = <function> defined @TipTac\ttCore.lua:813
 DISPLAY_SIZE_CHANGED = <function> defined @TipTac\ttCore.lua:550
 SetPaddingVariables = <function> defined @TipTac\ttCore.lua:660
 SetupPieceVisuals = <function> defined @SharedXML\Backdrop.lua:238
 backdropInfo = <table> {
 }
 TopRightCorner = <unnamed> {
 }
 SetBackdropColor = <function> defined @SharedXML\Backdrop.

**Comment by alphabitnz** [comment]
Same thing here after updating to 22.04.15

**Comment by alphabitnz** [comment]
Testing on retail gives just one error so far, tooltip is not as buggy as in classic but none of the text is coloured:

```
34x TipTac\ttCore.lua:1187: bad argument #1 to 'unpack' (table expected, got nil)
[string "=[C]"]: ?
[string "@TipTac\ttCore.lua"]:1187: in function `ApplyUnitAppearance'
[string "@TipTac\ttCore.lua"]:1409: in function <TipTac\ttCore.lua:1368>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@TheUndermineJournal\libs\LibExtraTip\LibExtraTip.lua"]:499: in function <...TheUndermineJournal\libs\LibExtraTip\LibExtraTip.lua:493>

Locals:
(*temporary) = nil
(*temporary) = "table expected, got nil"
```

**Comment by stevietv** [comment]
multiple errors on retail:

```
8x TipTac\ttCore.lua:1045: Usage: GameTooltip:SetPoint("point" [, region or nil] [, "relativePoint"] [, offsetX, offsetY])
[string "=[C]"]: in function `SetPoint'
[string "@TipTac\ttCore.lua"]:1045: in function <TipTac\ttCore.lua:1010>
[string "@TipTac\ttCore.lua"]:1472: in function <TipTac\ttCore.lua:1466>
[string "=[C]"]: ?
[string "=[C]"]: in function `GameTooltip_SetDefaultAnchor'
[string "@Bartender4\libs\LibActionButton-1.0\LibActionButton-1.0-82.lua"]:1299: in function <...er4\libs\LibActionButton-1.0\LibActionButton-1.0.lua:1296>
[string "@Bartender4\libs\LibActionButton-1.0\LibActionButton-1.0-82.lua"]:520: in function <...er4\libs\LibActionButton-1.0\LibActionButton-1.0.lua:518>

Locals:
(*temporary) = GameTooltip {
 0 = <userdata>
 SetQuestPartyProgress = <function> defined =[C]:-1
 SetText = <function> defined =[C]:-1
 OnBackdropLoaded = <function> defined @SharedXML\Backdrop.lua:144
 ItemTooltip = <unnamed> {
 }
 SetHyperlink = <function> defined =[C]:-1
 SetUnit = <function> defined =[C]:-1
 SetAzeriteEssence = <function> defined =[C]:-1
 RightEdge = <unnamed> {
 }
 SetVoidWithdrawalItem = <function> defined =[C]:-1
 ttSetBackdropBorderColorLocked = true
 SetSocketGem = <function> defined =[C]:-1
 ClearBackdrop = <function> defined @SharedXML\Backdrop.lua:274
 Show = <function> defined =[C]:-1
 SetPetAction = <function> defined =[C]:-1
 SetLFGDungeonReward = <function> defined =[C]:-1
 FadeOut = <function> defined =[C]:-1
 SetQuestLogSpecialItem = <function> defined =[C]:-1
 SetRecipeReagentItem = <function> defined =[C]:-1
 SetBuybackItem = <function> defined =[C]:-1
 SetWeeklyReward = <function> defined =[C]:-1
 SetMerchantItem = <function> defined =[C]:-1
 SetupTextureCoordinates = <function> defined @SharedXML\Backdrop.lua:206
 BottomOverlay = <unnamed> {
 }
 SetLootItem = <function> defined =[C]:-1
 SetQuestCurrency = <function> defined =[C]:-1
 ttIcon = <unnamed> {
 }
 GetEdgeSize = <function> defined @SharedXML\Backdrop.lua:180
 NineSlice = <unnamed> {
 }
 SetInventoryItem = <function> defined =[C]:-1
 TopEdge = <unnamed> {
 }
 ttCount = <unnamed> {
 }
 SetLootRollItem = <function> defined =[C]:-1
 SetTrainerService = <function> defined =[C]:-1
 SetExistingSocketGem = <function> defined =[C]:-1
 SetItemKey = <function> defined =[C]:-1
 SetBagItem = <function> defined =[C]:-1
 SetBackpackToken = <function> defined =[C]:-1
 BottomRightCorner = <unnamed> {
 }
 SetBackdropColor = <function> defined @TipTac\ttCore.lua:1656
 LeftEdge = <unnamed> {
 }
 SetMerchantCostItem = <function> defined =[C]:-1
 SetHeirloomByItemID = <function> defined =[C]:-1
 SetCurrencyByID = <function> defined =[C]:-1
 SetLootCurrency = <function> defined =[C]:-1
 ttSetIconTextureAndText = <function> defined @TipTacItemRef\ttItemRef.lua:169
 SetVoidDepositItem = <function> defined =[C]:-1
 SetSpellBookItem = <function> defined =[C]:-1
 SetVoidItem = <function> defined =[C]:-1
 textLeft1Font = "GameTooltipHeaderText"
 ARKTTD = <table> {
 }
 SetCompanionPet = <function> defined =[C]:-1
 SetCurrencyToken = <function> defined =[C]:-1
 SetGuildBankItem = <function> defined =[C]:-1
 updateTooltipTimer = 0.200000
 SetBackdropBorderColor = <function> defined @TipTac\ttCore.lua:1664
 HasBackdropInfo = <function> defined @SharedXML\Backdrop.lua:270
 Hide = <function> defined =[C]:-1
 SetCurrencyTokenByID = <function> defined =[C]:-1
 SetAzeriteEssenceSlot = <function> defined =[C]:-1
 SetEnhancedConduit = <function> defined =[C]:-1
 textRight2Font = "GameTooltipText"
 SetConduit = <function> defined =[C]:-1
 ClearLines = <function> defined =[C]:-1
 SetBackdrop = <function> defined @TipTac\ttCore.lua:1648
 SetAction = <function> defined =[C]:-1
 TopOverlay = <unnamed> {
 }
 GetBackdropBorderColor = <function> defined @SharedXML\Backdrop.lua:401
 GetBackdrop = <function> defined @SharedXML\Backdrop.lua:339
 SetUnitDebuff = <function> defined =[C]:-1
 SetItemByID = <function> defined =[C]:-1
 ApplyBackdrop = <function> defined @TipTac\ttCore.lua:1640
 SetQuestItem = <function> defined =[C]:-1
 Center = <unnamed> {
 }
 SetBorderBlendMode = <function> defined @SharedXML\Backdrop.lua:258
 SetToyByItemID = <function> defined =[C]:-1
 BottomEdge = <unnamed> {
 }
 BottomLeftCorner = <unnamed> {
 }
 TopRightCorner = <unnamed> {
 }
 SetInboxItem = <function> defined =[C]:-1
 TopLeftCorner = <unnamed> {
 }
 shoppingTooltips = <table> {
 }
 backdropInfo = <table> {
 }
 SetLFGDungeonShortageReward = <function> defined =[C]:-1
 SetQuestLogItem = <function> defined =[C]:-1
 OnBackdropSizeChanged = <function> defined @SharedXML\Backdrop.lua:174
 SetTradeTargetItem = <function> 
```

```
31x TipTac\ttCore.lua:661: attempt to perform arithmetic on field 'right' (a nil value)
[string "@TipTac\ttCore.lua"]:661: in function `SetPaddingVariables'
[string "@TipTac\ttCore.lua"]:1415: in function <TipTac\ttCore.lua:1413>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `SetOwner'
[string "@FrameXML\GameTooltip.lua"]:170: in function <FrameXML\GameTooltip.lua:169>
[string "=[C]"]: ?
[string "=[C]"]: in function `GameTooltip_SetDefaultAnchor'
[string "@Bartender4\libs\LibActionButton-1.0\LibActionButton-1.0-82.lua"]:1299: in function <...er4\libs\LibActionButton-1.0\LibActionButton-1.0.lua:1296>
[string "@Bartender4\libs\LibActionButton-1.0\LibActionButton-1.0-82.lua"]:749: in function <...er4\libs\LibActionButton-1.0\LibActionButton-1.0.lua:704>

Locals:
self = TipTac {
 BottomLeftCorner = <unnamed> {
 }
 close = <unnamed> {
 }
 AddModifiedTip = <function> defined @TipTac\ttCore.lua:2001
 MirrorAnchorsSmart = <table> {
 }
 CURSOR_UPDATE = <function> defined @TipTac\ttCore.lua:455
 GetNearestPixelSize = <function> defined @TipTac\ttCore.lua:607
 GetBackdropColor = <function> defined @SharedXML\Backdrop.lua:382
 RightEdge = <unnamed> {
 }
 CVAR_UPDATE = <function> defined @TipTac\ttCore.lua:534
 ClearBackdrop = <function> defined @SharedXML\Backdrop.lua:274
 SetBackdropColorLocked = <function> defined @TipTac\ttCore.lua:774
 ADDON_LOADED = <function> defined @TipTac\ttCore.lua:1910
 SetBackdropBorderColorLocked = <function> defined @TipTac\ttCore.lua:789
 ResetBackdropBorderColorLocked = <function> defined @TipTac\ttCore.lua:804
 tipsToModify = <table> {
 }
 SetupTextureCoordinates = <function> defined @SharedXML\Backdrop.lua:206
 SendElementEvent = <function> defined @TipTac\ttCore.lua:423
 HasBackdropInfo = <function> defined @SharedXML\Backdrop.lua:270
 GetEdgeSize = <function> defined @SharedXML\Backdrop.lua:180
 TopEdge = <unnamed> {
 }
 ApplyUnitAppearance = <function> defined @TipTac\ttCore.lua:1172
 BottomRightCorner = <unnamed> {
 }
 padding = <table> {
 }
 LeftEdge = <unnamed> {
 }
 UI_SCALE_CHANGED = <function> defined @TipTac\ttCore.lua:541
 SetBackdropLocked = <function> defined @TipTac\ttCore.lua:767
 SetBackdropBorderColor = <function> defined @SharedXML\Backdrop.lua:414
 ApplyHooksToTips = <function> defined @TipTac\ttCore.lua:1733
 BottomEdge = <unnamed> {
 }
 SetBackdrop = <function> defined @SharedXML\Backdrop.lua:321
 GetBackdropBorderColor = <function> defined @SharedXML\Backdrop.lua:401
 GetBackdrop = <function> defined @SharedXML\Backdrop.lua:339
 playerLevel = 60
 SetBorderBlendMode = <function> defined @SharedXML\Backdrop.lua:258
 isColorBlind = false
 MirrorAnchors = <table> {
 }
 RegisterElement = <function> defined @TipTac\ttCore.lua:417
 CreatePushArray = <function> defined @TipTac\ttCore.lua:397
 PLAYER_LEVEL_UP = <function> defined @TipTac\ttCore.lua:474
 RemoveUnwantedLines = <function> defined @TipTac\ttCore.lua:1081
 AnchorFrameToMouse = <function> defined @TipTac\ttCore.lua:1051
 elements = <table> {
 }
 ClassColorMarkup = <table> {
 }
 Center = <unnamed> {
 }
 OnBackdropSizeChanged = <function> defined @SharedXML\Backdrop.lua:174
 ApplyHooksToLFGLFAVSFB = <function> defined @TipTac\ttCore.lua:1877
 AddLockingFeature = <function> defined @TipTac\ttCore.lua:1630
 GetUnitReactionIndex = <function> defined @TipTac\ttCore.lua:1111
 OnBackdropLoaded = <function> defined @SharedXML\Backdrop.lua:144
 ApplySettings = <function> defined @TipTac\ttCore.lua:676
 ApplyHooksToCFMLLSF = <function> defined @TipTac\ttCore.lua:1854
 TopLeftCorner = <unnamed> {
 }
 0 = <userdata>
 text = <unnamed> {
 }
 AddTargetedBy = <function> defined @TipTac\ttCore.lua:1137
 ApplyTipBackdrop = <function> defined @TipTac\ttCore.lua:890
 ApplyBackdrop = <function> defined @SharedXML\Backdrop.lua:286
 ReApplyAnchorTypeForMouse = <function> defined @TipTac\ttCore.lua:1059
 SetPadding = <function> defined @TipTac\ttCore.lua:813
 DISPLAY_SIZE_CHANGED = <function> defined @TipTac\ttCore.lua:550
 SetPaddingVariables = <function> defined @TipTac\ttCore.lua:660
 SetupPieceVisuals = <function> defined @SharedXML\Backdrop.lua:238
 backdropInfo = <table> {
 }
 TopRightCorner = <unnamed> {
 }
 SetBackdropColor = <function> defined @SharedXML\Backdrop.
```

**Comment by frozn** [comment]
It's completely unclear to me how I could have missed this error. :confused:

Sorry for the inconvenience. I fixed this in the new release [v22.04.16](https://github.com/frozn/TipTac/releases/tag/v22.04.16). Thanks for reporting this! 👍
